docs/submitters/submissionGuide.md

```diff
@@ -0,0 +1,169 @@
+1. Convert an add-on to a json file:
+`python convertAddonManifest.py [addonId] [stable|beta|dev] [addonUrl]`
+	- `addonID` is the ID of the add-on. This should match the `name` field in the add-on manifest, e.g. "speechPlayer"
+	- `stable|beta|dev` are the channel options, choose one of these.
+	- `addonUrl` is where the addon file is hosted.
+1. Edit the auto-generated json file as required.
+
+## Filling out the template
+
+### addonId
+The ID for the addon.
+This should match the name field in the addon manifest and the folder name for the submission.
+The convention is a camelCaseName.
+
+Example: `easyAddonTech`
+
+### channel
+Should be `"stable", "beta" or "dev"`.
+
+**Stable:** Offered to users of any compatible version of NVDA.
+
+**Beta**: Allows add-on authors to target a stable NVDA release and get feedback from early adopter users before publishing add-on as stable.
+Offered to users of any compatible version of NVDA.
+
+**Dev**: Not offered to users of stable NVDA, only pre-release NVDA.
+This includes alpha, beta, rc, try, or dev builds.
+This channel can be used for developing add-ons against the newest unreleased additions to NVDA, or to stay up to date with compatibility breaking changes in an API breaking release cycle.
+
+### addonVersionNumber
+The version of the add-on, as a major-minor-patch dictionary.
+Add-on versions are expected to be unique for the addonId,
+meaning that a beta, stable and dev add-on version cannot share a version number.
+This is so there can be a unique ordering of newest to oldest.
+
+The suggested convention is to increment the patch version number for dev versions,
+increment the minor version number for beta versions,
+and increment the major version number for stable versions.
+
+Example:
+```json
+{
+	"major": 21,
+	"minor": 6,
+	"patch": 0
+}
+```
+
+### addonVersionName
+The addon version being released.
+Must match the version in the addon manifest and the file name for the submission.
+
+Example: `21.6.0`.
+
+### displayName
+The name that will be displayed in English for the addon.
+
+Example: `"Easy Addon"`
+
+### publisher
+The name of the individual, group, or company responsible for the addon.
+
+Example: `"NV Access"`
+
+### description
+The English description of the addon that will be displayed for the addon.
+	
+Example: `"Makes doing XYZ easier"`
+
+### homepage
+Optional.
+If the addon has a homepage where users can get more information about the addon, you can specify it here.
+
+Example: `"https://github.com/nvaccess/addon-datastore"`
+
+### minNVDAVersion
+The addon will not work with versions of NVDA prior to this version.
+
+Example:
+```json
+{
+	"major": 2021,
+	"minor": 1,
+	"patch": 0
+}
+```
+
+### lastTestedVersion
+The add-on has been tested up to and including this version of NVDA.
+
+Example:
+```json
+{
+	"major": 2020,
+	"minor": 4,
+	"patch": 0
+}
+```
+
+### URL
+To allow directly downloading the *.nvda-addon file.
+The URL should remain valid indefinitely.
```

Comment:
What happens if an URL is not available anymore? E.g. deleting a GitHub release by mistake.

From the add-on author point of view, I guess that this version should be revoked and a new should be made. But what if the author do not revoke it? May an add-on be revoked due to a permanently broken link (or a modified SHA)?

Maybe it's out of the scope for this first PR. If so, keep this remark for later work on a document dedicated to add-on revoking.